### ISSUE TITLE
Video?

### ISSUE THREAD

**Issue by ttseng** [miromannino/Justified-Gallery#75]
Can you add video (iframes and video tags)?

**Comment by miromannino** [comment]
No, this is specific for images. That can be done without any problem, but not now.

**Comment by miromannino** [comment]
That can be done now in 3.6 with the non-image entries.
@ttseng tell me if you have some problems with that, because it depends on the type of video.

**Comment by ametthey** [comment]
I have issue with 16:9 iframe, i doesn't respect the ratio correctly

**Comment by jcconnell** [comment]
@miromannino I'm trying to use mixed input formats - both video and images. I'm having some trouble getting YouTube videos to display in the gallery. Do you have an example of a video implementation with your gallery, preferably one that uses YouTube?

**Comment by acwolff** [comment]
See [this album with videos](
http://andrewolff.jalbum.net/TestVideos_FB).

**Comment by jcconnell** [comment]
@acwolff thanks for the help. Are you manually uploading video thumbnails when you add videos?

**Comment by acwolff** [comment]
No, videos are added via [jAlbums web locations](https://jalbum.net/help/en/Making_an_album_of_albums_with_web_locations) where it is important that you use the shared embed video link (“https://www.youtube.com/embed/...”).
